test/ServerComparison.TestSites/StartupNtlmAuthentication.cs

```diff
@@ -45,41 +47,41 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
                     }
                     else
                     {
-                        return context.Authentication.ChallengeAsync();
+                        return context.ChallengeAsync();
                     }
                 }
 
                 if (context.Request.Path.Equals("/Forbidden"))
                 {
-                    return context.Authentication.ForbidAsync(Microsoft.AspNetCore.Http.Authentication.AuthenticationManager.AutomaticScheme);
+                    return context.ForbidAsync();
                 }
 
                 if (context.Request.Path.Equals("/AutoForbid"))
                 {
-                    return context.Authentication.ChallengeAsync();
+                    return context.ChallengeAsync();
                 }
 
                 if (context.Request.Path.Equals("/RestrictedNegotiate"))
```

Comment:
We can just remove RestrictedNegotiate and RestrictedNTLM, they're not supported by the current design